lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -14,6 +14,8 @@
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE TypeOperators #-}
 
+{-# OPTIONS_GHC -fno-warn-redundant-constraints #-}
+
```

Comment:
> Which one are they?

I added this simply to avoid having to remove all the (currently unnecessary) constraints from Byron skeleton functions only to have to add them again when we fill in the function definitions (which will then require the constraints again).